x/tokenomics/keeper/keeper_settle_pending_claims_test.go

```diff
@@ -275,7 +275,7 @@ func (s *TestSuite) TestClaimSettlement_ClaimSettled_ProofRequiredAndProvided_Vi
 	// NB: proof window has definitely closed at this point
 	blockHeight := shared.GetProofWindowCloseHeight(&sharedParams, claim.SessionHeader.SessionEndBlockHeight)
 	sdkCtx = sdkCtx.WithBlockHeight(blockHeight)
-	numClaimsSettled, numClaimsExpired, err := s.keepers.SettlePendingClaims(sdkCtx)
+	numClaimsSettled, numClaimsExpired, _, err := s.keepers.SettlePendingClaims(sdkCtx)
```

Comment:
Merge w/ main? Kind of weird we're seeing these changes here.

Reply:
The need to make these changes was the result of merging with main at the time.